environment_setup.sh

```diff
@@ -88,38 +88,30 @@ if confirmupdate "Would you like to install local development programs like PHPS
   echo "Installing alternate PHP versions (5.6, 7.1)"
   echo $'\n'
 
-  brew unlink php70
-
-  brew install php56
-  brew install php56-opcache
-  brew install php56-mcrypt
-  brew install php56-xdebug
-  brew install php56-redis
-
-  brew unlink php56
-
-  brew install php71
-  brew install php71-opcache
-  brew install php71-mcrypt
-  brew install php71-xdebug
-  brew install php71-redis
-
-  brew unlink php71
-  brew link php70
+  brew install php@56
+  brew install php@70
+  brew install php@71
 
   echo $'\n'
   echo "Configuring PHP"
   echo $'\n'
 
-  for VER in 5.6 7.0 7.1
+  echo $'\n'
+  echo "Installing xdebug, redis, and mcrypt with pecl"
+  echo $'\n'
+  pecl install xdebug
+  pecl install redis
+  pecl install mcrypt
```

Comment:
I believe we need to do this for each version of php - right? And the old versions of php are keg-only so when they're linked they don't appear in your path unless you force them into your path... makes installing pecl extensions for old versions of php pretty annoying